packages/jest-create-cache-key-function/src/index.ts

```diff
@@ -40,7 +40,15 @@ type NewGetCacheKeyFunction = (
 
 type GetCacheKeyFunction = OldGetCacheKeyFunction | NewGetCacheKeyFunction;
 
-function getGlobalCacheKey(files: Array<string>, values: Array<string>) {
+/**
+ * Computes the global cache key given  a collection of files and values.  This limits
+ * the output with a provided length.
+ * @param files list of files to read
+ * @param values list of values to add to the computation
+ * @param length length of the resulting key defaults to 16
```

Comment:
need to pass this in `createCacheKey` as well

Reply:
What do you mean? Where is createCacheKey?

Reply:
It's the exported function